src/config/graphical-session/graphics-drivers/index.md

```diff
@@ -5,6 +5,7 @@ of your system.
 
 ## Section Contents
 
-- [Intel GPU](./intel.md)
-- [NVIDIA Optimus](./optimus.md)
+- [AMD](./amd.md)
+- [Intel](./intel.md)
 - [NVIDIA](./nvidia.md)
+- [NVIDIA Optimus](./optimus.md)
```

Comment:
I like this organization, but the changes should be made in the SUMMARY as well.

Reply:
Ideally the title here matches the one in the page, too (I believe it is necessary for the PDF gen). Perhaps "AMD and ATI" ?